src/components/Header/Navbar.jsx

```diff
@@ -19,29 +14,28 @@ export const Navbar = () => {
       fontSize="xl"
       color={textColor}
       fontWeight="bold"
+      sx={{ ol: { display: 'flex', alignItems: 'center' } }}
     >
-      <Flex direction={{ base: 'column', lg: 'row', xl: 'row' }}>
-        <BreadcrumbItem mr={6} mb={{ base: 4, lg: 0 }} mt={{ base: 6, lg: 0 }}>
-          <BreadcrumbLink as={Link} to="/" textUnderlineOffset="4px">
-            {i18next.t('header.navigation.home')}
-          </BreadcrumbLink>
-        </BreadcrumbItem>
-        <BreadcrumbItem mr={6} mb={{ base: 4, lg: 0 }}>
-          <BreadcrumbLink as={Link} to="/incomes">
-            {i18next.t('header.navigation.incomes')}
-          </BreadcrumbLink>
-        </BreadcrumbItem>
-        <BreadcrumbItem mr={6} mb={{ base: 4, lg: 0 }}>
-          <BreadcrumbLink as={Link} to="/categories" textUnderlineOffset="4px">
-            {i18next.t('header.navigation.categories')}
-          </BreadcrumbLink>
-        </BreadcrumbItem>
-        <BreadcrumbItem mr={6} mb={{ base: 4, lg: 0 }}>
-          <BreadcrumbLink as={Link} to="/expenses" textUnderlineOffset="4px">
-            {i18next.t('header.navigation.expenses')}
-          </BreadcrumbLink>
-        </BreadcrumbItem>
-      </Flex>
+      <BreadcrumbItem mr={{ base: 2, xl: 4 }}>
+        <BreadcrumbLink as={Link} to="/" textUnderlineOffset="4px">
+          {i18next.t('header.navigation.home')}
+        </BreadcrumbLink>
+      </BreadcrumbItem>
+      <BreadcrumbItem mr={{ base: 2, xl: 4 }}>
+        <BreadcrumbLink as={Link} to="/incomes">
```

Comment:
```suggestion
        <BreadcrumbLink as={Link} to="/incomes" textUnderlineOffset="4px">
```

I think it was missed somehow)